templates/channels/channel_configuration.haml

```diff
@@ -432,6 +432,51 @@
     %code
       https://{{ domain }}{% url 'handlers.hub9_handler' 'delivered' object.uuid %}
 
+  -elif object.channel_type == 'FCM'
+    %h4
+      -trans "To finish configuring your Firebase Cloud Messaging connection you'll need to send the data to URL with parameters `from`, `msg` and `fcm_token` to receive messages and to register contacts use `urn`, `fcm_token` and `name` (optional)."
```

Comment:
To use your Firebase Cloud Messaging channel you'll have to POST to the following URLs with the parameters below.